tests/integration/tier0/non-destructive/subscription-manager/main.fmf

```diff
@@ -90,3 +90,31 @@ tag+:
             pytest -svv -m test_unregistered_no_credentials
         tag+:
             - unregistered-no-credentials
+    /no_sca_not_subscribed:
+        summary+: |
+            The subscription is auto-attached on a pre-registered system
```

Comment:
```suggestion
            The subscription is auto-attached by the convert2rhel utility on a pre-registered system
```

Reply:
I will try to reword this, but I'd still like to keep this around 50 characters :D